com.overlayed.dev.yaml

```diff
@@ -0,0 +1,129 @@
+        make-install-args:
+          - prefix=${FLATPAK_DEST}
+        sources:
+          - type: archive
+            url: https://dotat.at/prog/unifdef/unifdef-2.12.tar.xz
+            sha256: 43ce0f02ecdcdc723b2475575563ddb192e988c886d368260bc0a63aee3ac400
+        cleanup:
+          - "*"
+
+  - name: openssl11
+    buildsystem: simple
+    build-commands:
+      - ./config --prefix=${FLATPAK_DEST}
+      - make -j $FLATPAK_BUILDER_N_JOBS
+      - make install_sw
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - "*.a"
+      - "*.la"
+    sources:
+      - type: archive
+        url: https://www.openssl.org/source/openssl-1.1.1w.tar.gz
+        sha256: cf3098950cb4d853ad95c0841f1f9c6d3dc102dccfcacd521d93925208b76ac8
+        x-checker-data:
+          type: anitya
+          project-id: 20333
+          url-template: https://www.openssl.org/source/openssl-$version.tar.gz
+
+  # for the in-game narrator
+  - name: flite
+    buildsystem: autotools
+    config-opts:
+      - --enable-shared
+    no-parallel-make: true
+    cleanup:
+      - /bin
+      - /include
+      - "*.a"
+    sources:
+      - type: git
+        url: https://github.com/festvox/flite.git
+        tag: v2.2
+        commit: e9e2e37c329dbe98bfeb27a1828ef9a71fa84f88
+        x-checker-data:
+          type: git
+          tag-pattern: ^v([\\d.]+)$
+
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+
+  # for some controller mods (see https://github.com/isXander/Controlify/issues/31)
+  - shared-modules/libusb/libusb.json
+
+  # TODO: compile from source (preferably before the launcher supports arm on linux)
+  - name: overlayed
+    buildsystem: simple
+    build-commands:
+      - chmod +x Overlayed.AppImage
+      - ./Overlayed.AppImage --appimage-extract
+
+      - |
+        desktop-file-edit \
+          --set-name="Overlayed App" \
+          --set-comment="Overlayed's game launcher" \
+          --set-icon="$FLATPAK_ID" \
+          --set-key="StartupNotify" --set-value="true" \
+          --set-key="Categories" --set-value="Game;ActionGame;AdventureGame;Simulation;" \
+          --set-key="Keywords" --set-value="game;minecraft;mc;" \
+          --set-key="StartupWMClass" --set-value="Overlayed" \
+          squashfs-root/usr/share/applications/overlayed.desktop
+
+      - mkdir -p /app/{bin,share}
+      - cp -r squashfs-root/usr/share/icons /app/share/icons
+      - install -Dm755 squashfs-root/usr/bin/overlayed /app/bin/overlayed-app
+      - install -Dm644 squashfs-root/usr/share/applications/overlayed.desktop /app/share/applications/com.overlayed.Overlayed.desktop
+      - install -Dm644 com.overlayed.Overlayed.metainfo.xml /app/share/metainfo/com.overlayed.OverlayedApp.metainfo.xml
+
+    sources:
+      - type: file
+        path: com.overlayed.Overlayed.metainfo.xml
+
+      - type: file
+        url: https://github.com/overlayeddev/overlayed/releases/download/v0.5.0/overlayed_0.5.0_amd64.AppImage
```

Comment:
Why not build from source?

Reply:
If you can point me to any other tauri app building from source I can look into it.

Reply:
> If you can point me to any other tauri app building from source I can look into it.

What version of tauri are you using? If it is not v2 then that explains including webkit2gtk



https://github.com/flathub/in.cinny.Cinny/blob/master/in.cinny.Cinny.yml is a tauri app that builds from source.

Reply:
openssl1 is likely not needed. You can remove that.